inspirehep/config.py

```diff
@@ -419,6 +419,30 @@ def _(x):
         read_permission_factory_imp="inspirehep.modules.records.permissions:record_read_permission_factory",
         record_class='inspirehep.modules.records.es_record:ESRecord'
     ),
+    literature_citesummary=dict(
+        default_media_type='application/json',
+        item_route=(
+            '/literature'
+            '/<pid(literature,record_class="inspirehep.modules.records.es_record:ESRecord"):pid_value>'
+            '/citesummary'),
+        list_route='/literature/',
+        max_result_window=10000,
```

Comment:
This might mean that we under report on exactly the two papers with more than 10k citations. I'm not sure if we can push this number a little higher and fix this issue, or we would have to live with that...